packages/cdk/lib/roles/cromwell-adapter-role.ts

```diff
@@ -0,0 +1,19 @@
+import * as cdk from "monocdk";
+import * as iam from "monocdk/aws-iam";
+import { BucketOperations } from "../../common/BucketOperations";
+
+export interface CromwellAdapterRoleProps {
+  readOnlyBucketArns: string[];
+  readWriteBucketArns: string[];
+}
+
+export class CromwellAdapterRole extends iam.Role {
+  constructor(scope: cdk.Construct, id: string, props: CromwellAdapterRoleProps) {
+    super(scope, id, {
+      assumedBy: new iam.ServicePrincipal("ecs-tasks.amazonaws.com"),
```

Comment:
Presumably this would be set by default if we just left the taskRole alone.

Any reason to externalize this role entirely rather than do something like `buckets.grantReadAccess(adapter)`

Reply:
Just for sake of consistency between the code. Seemed a little odd to have nextflow's role set its own s3 permissions while we use the cromwell-engine-stack to do that for cromwell.